src/todos.js

```diff
@@ -1,48 +1,48 @@
 export default class Todos {
-    constructor() {
-      this.list = localStorage.getItem('todos')
-        ? JSON.parse(localStorage.getItem('todos'))
-        : [];
-    }
-  
-    setLocalStorage() {
-      localStorage.setItem('todos', JSON.stringify(this.list));
-    }
-  
-    addTodo(todo) {
-      this.list.push(todo);
-      this.setLocalStorage();
-    }
-  
-    removeTodo(todoID) {
-      this.list = this.list.filter((todo) => todo.id !== todoID);
-      this.list.forEach((todo, index) => {
-        todo.index = index + 1;
-      });
-      this.setLocalStorage();
-    }
-  
-    editTodo(todoId, todoDescription) {
-      this.list = this.list.map((todo) => {
-        if (todo.id === todoId) {
-          return { ...todo, description: todoDescription };
-        }
-        return todo;
-      });
-      this.setLocalStorage();
-    }
-  
-    completeTodo(todoId, status) {
-      const selected = this.list.findIndex((element) => element.id === todoId);
-      this.list[selected].completed = status;
-      this.setLocalStorage();
-    }
-  
-    clearCompletedTodos() {
-      this.list = this.list.filter((todo) => !todo.completed);
-      this.list.forEach((todo, index) => {
-        todo.index = index + 1;
-      });
-      this.setLocalStorage();
-    }
-  }
\ No newline at end of file
+  constructor() {
+    this.list = localStorage.getItem('todos')
+      ? JSON.parse(localStorage.getItem('todos'))
+      : [];
+  }
+
+  setLocalStorage() {
+    localStorage.setItem('todos', JSON.stringify(this.list));
+  }
+
+  addTodo(todo) {
+    this.list.push(todo);
+    this.setLocalStorage();
+  }
+
+  removeTodo(todoID) {
+    this.list = this.list.filter((todo) => todo.id !== todoID);
+    this.list.forEach((todo, index) => {
+      todo.index = index + 1;
+    });
+    this.setLocalStorage();
+  }
+
+  editTodo(todoId, todoDescription) {
+    this.list = this.list.map((todo) => {
+      if (todo.id === todoId) {
+        return { ...todo, description: todoDescription };
+      }
+      return todo;
+    });
+    this.setLocalStorage();
+  }
+
+  completeTodo(todoId, status) {
+    const selected = this.list.findIndex((element) => element.id === todoId);
+    this.list[selected].completed = status;
+    this.setLocalStorage();
+  }
+
+  clearCompletedTodos() {
+    this.list = this.list.filter((todo) => !todo.completed);
+    this.list.forEach((todo, index) => {
+      todo.index = index + 1;
+    });
+    this.setLocalStorage();
+  }
+}
```

Comment:
### PROJECT DESCRIPTION

- Hey, in this project milestone, you're meant to create an array with a  list of tasks and create a function to populate the created task in the page as the page loads. however, you've written a lot of functions for feature project milestones, this main project requirement has not been implemented yet. please, refer to the project requirement to fix this issue.

![image](https://user-images.githubusercontent.com/59397159/236116851-f66e2ef9-ada7-49cc-9b6d-a0d6d75e45a3.png)